src/@types/types.d.ts

```diff
@@ -0,0 +1,2 @@
+//e.g. 1.2.0 is described as [1, 2, 0]
+export type SemanticVersion = [number, number, number]
```

Comment:
Added new type 'SemanticVersion' to describe versions as an array of numbers(referred to WPD json schema)